ipfs-pinning-service.yaml

```diff
@@ -461,6 +462,14 @@ components:
       example:
         status_details: "Queue position: 7 of 9" # PinStatus.info[status_details], when status=queued
 
+    TextMatchingStrategy:
+      description: Alternative text matching strategy
+      type: string
+      default: exact
+      enum:
+        - exact   # full match, case-sensitive (the implicit default)
+        - fuzzy   # partial match, case-insensitive
```

Comment:
:bike: + :derelict_house: :
"fuzzy" I like that it's short, but not so happy that it's not quite precise in what it does. I'm fine with fuzzy, but putting out the call for better names before we decide we're just going with this one.

Reply:
I am totally fine with renaming `fuzzy` to something better. 
Food for thought: `partial`, `relaxed`, `inexact`,  `loose`

Reply:
+1 for `partial` unless you specifically need to indicate it's case-insensitive too (if the latter, would suggest something like `full-sensitive` and `partial-insensitive` but that gets a little too bikesheddy, probably)

Reply:
+1 for `partial`

Reply:
Renamed to `partial` in [`91299cc` (#66)](https://github.com/ipfs/pinning-services-api-spec/pull/66/commits/91299ccbdba9b1a6808d58ab828ecc2772b9efdf).

We need to figure out if thats enough, or do we want something more fancy like suggestion from https://github.com/ipfs/pinning-services-api-spec/pull/66#issuecomment-712134357